### PR TITLE
Docs: [DS-212] [DS-216] [DS-214] PasswordField, TextField & SearchField

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: apps/docs/content/components/forms/PasswordField.mdx [hunk 1]
```diff
@@ -1,48 +1,89 @@
...
+
+<Example src="inputs/docs/passwordField/readOnly" />
+
+### Error
+A password field with an error message.
+
+<Example src="inputs/docs/passwordField/error" />
+
+### Sizes
+Password fields come in two sizes.
```
Comment (fraincs) [inline apps/docs/content/components/forms/PasswordField.mdx:62]: I would be more generic, like in other size component docs, this way if we add a third size, nothing to change here :D

File: apps/docs/content/components/forms/SearchField.mdx [hunk 2]
```diff
@@ -1,48 +1,94 @@
...
+
+<Example src="inputs/docs/searchField/error" />
+
+### Hide Clear Button
+A search field can hide its clear button.
+
+<Example src="inputs/docs/searchField/hideClearButton" />
+
+### Sizes
+Search fields come in two sizes.
```
Comment (fraincs) [inline apps/docs/content/components/forms/SearchField.mdx:67]: Same comment as above

File: apps/docs/content/components/forms/TextField.mdx [hunk 3]
```diff
@@ -1,48 +1,115 @@
...
+
+<Example src="inputs/docs/textField/error" />
+
+### Clearable
+The `isClearable` prop can be set to `true` to display a clear button at the end of the input.
+
+<Example src="inputs/docs/textField/clearable" />
+
+### Sizes
+Search fields come in two sizes.
```
Comment (fraincs) [inline apps/docs/content/components/forms/TextField.mdx:71]: Same comment

File: packages/components/src/inputs/docs/passwordField/description.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { PasswordField, Label, HelperMessage } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <PasswordField placeholder="Placeholder">
+            <Label>Password:</Label>
+            <HelperMessage>Helper message</HelperMessage>
```
Comment (fraincs) [inline packages/components/src/inputs/docs/passwordField/description.tsx:7]: I would change this for a hint such as : more than 10 characters.

Comment (vicky-comeau) [reply packages/components/src/inputs/docs/passwordField/description.tsx:7]: true, I did passwordField first and forgot to go back and put "real life" text

**Review by fraincs — APPROVED** [review]

**Pull request merged**